### ISSUE TITLE
Loko Air BOM

### ISSUE THREAD

**Issue by tyagodani** [tomipiriyev/Loko#8]
Will the BOM for Air be made available?

**Comment by tomipiriyev** [comment]
@tyagodani   It is added now, forgotten

**Comment by tyagodani** [comment]
@tomipiriyev thanks a lot!
By the way, been looking at all the parts and ti seems that the PA1010D GPS module has been discontinued. Really can't seem to find it anywhere. Is there another module that came to replace this one? Or where do you get it from? Cheers

**Comment by Stokestack** [comment]
This one has Lora and GPS integrated: https://www.seeedstudio.com/Wio-WM1110-Module-LR1110-and-nRF52840-p-5676.html

Seems like the project could be revised to use that, no?

**Comment by tomipiriyev** [comment]
@tyagodani   gps you can find from https://www.mirifica.de/en/cd-top/mediatek-mt3333/cdtop-technology-pa1010dv2-cd-pa1010d2_100011_1327.  

LR1110 can't be used, it does processing on LoraCLound , not in the mcu to save power

**Comment by tyagodani** [comment]
Cool, thanks @tomipiriyev! One more question: on the link you sent there are two version. "Version 1" and "GPS+GLONASS". Which one should I wonder, if there is even a difference?
Thank you

**Comment by tomipiriyev** [comment]
@tyagodani  yes it has differences,   it is better to use GPS+Galileo+Glonass,  to have faster search . but I haven't made hands on comparison of these two versions . we use v1

**Comment by tyagodani** [comment]
@tomipiriyev thanks for the briefing. Already ordered!
One last thing, if you wouldn't mind. I'm thinking of ordering the PCB with the smaller SMD components already assembled and I'll solder things like the GPS and Lora modules and the port myself (I still don't have the proper equipment to solder the small SMD stuff nor the practice). Could you please upload the CPL Pick&Place file?
Thank you once again!

**Comment by tomipiriyev** [comment]
@tyagodani  uploaded

**Comment by tyagodani** [comment]
Thank you and thanks for the tips!

**Comment by marksdemellin** [comment]
Hello, I wanted to ask about GPS module if it's possible to use the Quectel L86 since is easier to find and also cheaper. From technical parameters, seems to have the same ones. Maybe for the programming part, firmware need to be adapted, but I think it's not a big problem. What you think?

**Comment by tomipiriyev** [comment]
hi @marksdemellin  Quectel L86  is 4 times bigger than the current module we use. 

We are also looking for low price Chinese alternatives to use , to lower the production cost. 

incase if I find something suitable , I will notify you .  currently the size , weight , power consumption's are the main criteria for such a small product.
